dojo/tools/jfrog_xray_api_summary_artifact/parser.py

```diff
@@ -74,22 +69,20 @@ def get_item(vulnerability, service, test):
     if len(impact_paths) > 0:
         impact_path = decode_impact_path(impact_paths[0])
 
-    # The unique_id_from_tool is set only when a given component (SHA) has a specific unique Finding (XRAY or CVE)
     if 'issue_id' in vulnerability:
-        title = vulnerability['issue_id'] + " - " + impact_path.name + ":" + impact_path.version
-        unique_id_from_tool = vulnerability['issue_id'] + " " + impact_path.sha
+        unique_id_from_tool = str(hash(impact_path.name + impact_path.version + vulnerability['issue_id']))
+        vuln_id_from_tool = vulnerability['issue_id']
     elif cve:
-        title = str(cve) + " - " + impact_path.name + ":" + impact_path.version
-        unique_id_from_tool = str(cve) + " " + impact_path.sha
+        unique_id_from_tool = str(hash(impact_path.name + impact_path.version + cve))
     else:
-        title = impact_path.name + ":" + impact_path.version
-        unique_id_from_tool = None
+        unique_id_from_tool = str(hash(impact_path.name + impact_path.version + vulnerability['summary']))
+        vuln_id_from_tool = ""
 
     finding = Finding(
+        vuln_id_from_tool=vuln_id_from_tool,
         service=service,
-        title=title,
+        title=vulnerability['summary'],
```

Comment:
Now that the XRAY and CVEs are persisted in the right locations the titles were changed to be easier to read